server/routes/postQueryAnalysis.js

```diff
@@ -0,0 +1,47 @@
+const stopwordParser = require('../services/Parser')
+const parseIt = new stopwordParser()
+const express = require('express');
+const router = express.Router()
+
+let regex = /[^0-9a-z\s]/ig
+
+router.post('/', (req, res) => {
+    try{
+    let { query } = req.query
+    let queryArr = query.replace(regex, '').split(' ')
+    let data = []
+
+
+    for(let i = 0; i < queryArr.length; i++) {
+       let dateVal = parseIt.getDate(queryArr[i])
+       let mealtimeVal = parseIt.getMealtime(queryArr[i])
+       let stopwordVal = parseIt.getStopword(queryArr[i])
+       
+        if(dateVal.length > 0){
+            let dateValues = dateVal[0]
```

Comment:
We want all matches.

Reply:
But it will grab one match per entry. I will adjust.